subprojects/plugin-development/src/main/java/org/gradle/plugin/devel/plugins/IvyPluginPublishingPlugin.java

```diff
@@ -62,9 +62,6 @@ private void configurePublishing(final Project project) {
             @Override
             public void execute(PublishingExtension publishing) {
                 final GradlePluginDevelopmentExtension pluginDevelopment = project.getExtensions().getByType(GradlePluginDevelopmentExtension.class);
-                if (!pluginDevelopment.isAutomatedPublishing()) {
```

Comment:
I do not think we can just do that, it would impact anyone, including those using the plugin publish plugin aith a version before 1.0